CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+### 6.0.0
+
+* [UPDATE] Update target frameworks: .NET8, .NET Standard 2.0
+* [UPDATE] Drop EOL .NET 6/7 platforms from testing matrix
+* [UPDATE] Update github actions steps versions
+* [UPDATE] Remove legacy obsolete API
+* [UPDATE] Mark as obsolete api CompatArg with pre c# 7.0 support
```

Comment:
**thought:** For upcoming 5.3 release I used GitHub's "generate release notes" at the time of release to fill this in. Do you think that would be easier for contributors? (I generally enjoy going through all the changes and describing them and thanking contributors, but it is probably easier for everyone to generate this 🤔 )

Reply:
Yea, generate release notes works good as for me

There is small problem is that ```1 commit != 1 change```
+ We can add additionally short summary to inform consumer about some important changes  to release description if needed